### PR TITLE
MULE-16664: Create or adjust a job for testing extensions on Windows

### PR DESCRIPTION
**Pull request by lvzh90**

### CODE REVIEW

File: src/test/java/org/mule/extension/compression/GzipTestCase.java [hunk 1]
```diff
@@ -6,27 +6,26 @@
...
 import org.mule.runtime.core.api.util.IOUtils;
 import org.mule.runtime.extension.api.runtime.operation.Result;
 
 import java.io.ByteArrayInputStream;
 import java.io.InputStream;
 
+import static java.lang.Thread.currentThread;
+import static org.hamcrest.Matchers.*;
+import static org.junit.Assert.assertThat;
+import static org.mule.extension.compression.CompressionModuleTestUtils.*;
```
Comment (aiannucci) [inline src/test/java/org/mule/extension/compression/GzipTestCase.java]: Don't use wildcards and fix import ordering

File: src/test/java/org/mule/extension/compression/ZipStrategyTestCase.java [hunk 2]
```diff
@@ -31,16 +22,17 @@
...
-import org.hamcrest.collection.IsMapContaining;
-import org.junit.Rule;
-import org.junit.Test;
-import org.junit.rules.ExpectedException;
+import static java.lang.Thread.currentThread;
+import static org.hamcrest.Matchers.*;
+import static org.junit.Assert.assertThat;
+import static org.mule.extension.compression.CompressionModuleTestUtils.*;
+import static org.mule.runtime.api.metadata.DataType.INPUT_STREAM;
+import static org.mule.runtime.api.metadata.DataType.TEXT_STRING;
```
Comment (aiannucci) [inline src/test/java/org/mule/extension/compression/ZipStrategyTestCase.java]: Same as before

File: src/test/java/org/mule/extension/compression/GzipTestCase.java (hunk 1 above)
Comment (lvzh90) [reply src/test/java/org/mule/extension/compression/GzipTestCase.java]: Done

File: src/test/java/org/mule/extension/compression/ZipStrategyTestCase.java (hunk 2 above)
Comment (lvzh90) [reply src/test/java/org/mule/extension/compression/ZipStrategyTestCase.java]: Done

File: src/test/java/org/mule/extension/compression/GzipTestCase.java [hunk 3]
```diff
@@ -7,20 +7,25 @@
...
 import static org.junit.Assert.assertThat;
-import static org.mule.extension.compression.CompressionModuleTestUtils.*;
+import static org.mule.extension.compression.CompressionModuleTestUtils.DATA_SIZE;
+import static org.mule.extension.compression.CompressionModuleTestUtils.FILE_TXT_DATA;
+import static org.mule.extension.compression.CompressionModuleTestUtils.FILE_TXT_NAME;
+import static org.mule.extension.compression.CompressionModuleTestUtils.TEST_DATA;
 import static org.mule.runtime.api.metadata.DataType.TEXT_STRING;
 import static org.mule.runtime.core.api.util.IOUtils.toByteArray;
 
+import org.junit.Rule;
```
[the remark this answers is not in the document]
Comment (lvzh90) [reply src/test/java/org/mule/extension/compression/GzipTestCase.java]: Done

File: src/test/java/org/mule/extension/compression/ZipStrategyTestCase.java [hunk 4]
```diff
@@ -7,19 +7,25 @@
...
 import static org.mule.extension.compression.CompressionModuleTestUtils.DATA_SIZE;
 import static org.mule.extension.compression.CompressionModuleTestUtils.FILE_TXT_DATA;
 import static org.mule.extension.compression.CompressionModuleTestUtils.FILE_TXT_NAME;
 import static org.mule.extension.compression.CompressionModuleTestUtils.TEST_DATA;
-import static org.mule.extension.compression.CompressionModuleTestUtils.asTextTypedValue;
 import static org.mule.runtime.api.metadata.DataType.INPUT_STREAM;
 import static org.mule.runtime.api.metadata.DataType.TEXT_STRING;
 import static org.mule.runtime.core.api.util.IOUtils.toByteArray;
+
+import com.google.common.collect.ImmutableMap;
```
[the remark this answers is not in the document]
Comment (lvzh90) [reply src/test/java/org/mule/extension/compression/ZipStrategyTestCase.java]: Done

**Pull request merged**